upa/src/sdk/submissionIntervals.ts

```diff
@@ -169,7 +169,14 @@ export function mergeSubmissionIntervals<T>(
                     ${JSONstringify(curInterval)}`;
       }
 
-      if (nextInterval.startIdx !== 0) {
+      const curMetadata = curInterval.data as OffChainSubmissionMetadata;
+      const nextMetadata = nextInterval.data as OffChainSubmissionMetadata;
+
+      if (
```

Comment:
Logic makes sense. WOuld just explain why onchain sub. can be split across agg. batches and then startIdx is no longer 0 for second one.